bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -583,13 +583,8 @@ private String getMergeReadyComment(String commitMessage, List<Review> reviews)
         if (!censusInstance.isCommitter(pr.author())) {
             message.append("\n");
             var contributor = censusInstance.namespace().get(pr.author().id());
-            if (contributor == null) {
-                message.append("As you are not a known OpenJDK [Author](https://openjdk.java.net/bylaws#author), ");
-            } else {
-                message.append("As you do not have Committer status in this project, ");
-            }
-
-            message.append("an existing [Committer](https://openjdk.java.net/bylaws#committer) must agree to ");
+            message.append("As you do not have [Committer](https://openjdk.java.net/bylaws#committer) status in this project, ");
```

Comment:
Can we make `project` a link to the project in https://openjdk.java.net/census ?